tasks/compile_otp_plugin.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Load shared compilation variables
+  include_vars: "../vars/compilation.yml"
+
+- name: Install gcc objc repo
+  yum_repository:
+    name: csi-gcc
+    description: gcc compiler suite, with Objective-C which is removed from official Red Hat EL8 releases.
+    baseurl: "{{ gcc_objc_repo.base_url }}"
+    gpgkey: "{{ gcc_objc_repo.key }}"
+    gpgcheck: yes
+    enabled: yes
+
+- name: Install dev packages
+  package:
+   name: "{{ compile_develop_packages }}"
+   state: latest
+
+- name: Checkout otp repository
+  git:
+    repo: https://github.com/evgeny-gridasov/openvpn-otp
+    version: "{{ openvpn_auth_otp_version }}"
+    dest: "{{ compile_source_dir }}/openvpn-otp"
+
+- name: Build otp plugin
+  shell:
+    chdir: "{{ compile_source_dir }}/openvpn-otp"
+    creates: "{{ openvpn_plugin_dir_path }}/openvpn-otp.so"
+    cmd: |
+      ./autogen.sh
+      ./configure --with-openvpn-plugin-dir="{{ openvpn_plugin_dir_path }}"
+      make install
```

Comment:
This should all be handled by the proper tool - a build system. https://copr.fedorainfracloud.org/ is free & set up the repos.

Reply:
I disagree on that point. But feel free to close this PR as we have already moved onto using our own fork of this repository.

Reply:
There might be something I don't know/am missing - why do you think it's better to have the playbook handle the compilation steps? 
My view is that the compilation steps are more invasive (more packages, hardcoded assumptions) that ansible isn't set up to handle well.

I like the idea of having OTP as a plugin, and I am hitting issues with the LDAP plugin compiling on copr (#154), so understanding your view could be instructive for me. 

If you want to fix the conflicts (and add OTP as a [community feature](https://github.com/kyl191/ansible-role-openvpn#support-notesexpectations) I'm fine add merging this, and having the "move compilation out" as a follow up on my part.